src/igraph/__init__.py

```diff
@@ -382,30 +382,35 @@ def delete_edges(self, *args, **kwds):
         """Deletes some edges from the graph.
 
         The set of edges to be deleted is determined by the positional and
-        keyword arguments. If any keyword argument is present, or the
-        first positional argument is callable, an edge
-        sequence is derived by calling L{EdgeSeq.select} with the same
-        positional and keyword arguments. Edges in the derived edge sequence
-        will be removed. Otherwise the first positional argument is considered
-        as follows:
+        keyword arguments. If the functions is called without any arguments,
+        all edges are deleted. If any keyword argument is present, or the
+        first positional argument is callable, an edge sequence is derived by
+        calling L{EdgeSeq.select} with the same positional and keyword
+        arguments. Edges in the derived edge sequence will be removed.
+        Otherwise the first positional argument is considered as follows:
 
           - C{None} - deletes all edges
+            @deprecated: replaced by no arguments since igraph 0.8.3
```

Comment:
Not sure if Epydoc likes the `@deprecated` directive in the middle of a list. I would say just spell it out explicitly in a sentence.

Reply:
done